spec/source_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../classes/source'
+
+describe Source do
+  before :each do
+    @source = Source.new('Internet')
+  end
+
+  it 'Test if object is instance of correct class' do
+    expect(@source).to be_an_instance_of Source
+  end
+
+  it 'Source name should be correct' do
+    expect(@source.name).to eq('Internet')
+  end
+
+  it 'The add_items method should add item to @@items' do
+    item1 = Item.new(Date.today.to_s, true)
+    @source.add_item(item1)
+    items = Source.class_variable_get(:@@items)
+    expect(items).to include(item1)
+  end
+end
```

Comment:
- [ ]  Good job adding this test. However, this test is failing, let's try to fix it.

<img width="496" alt="chrome_c3Z8U5C7of" src="https://user-images.githubusercontent.com/81468675/154714128-fda851e2-bf5c-4bfb-a664-3170b8ba0c5c.png">